docker-compose.yml

```diff
@@ -2,31 +2,31 @@ version: '3'
 services:
     mongo-1-1:
         container_name: "mongo-1-1"
-        image: mongo:4.0.1
+        image: ${MONGODB_FLAVOR:-mongo}:${VERSION:-4.0.1}
```

Comment:
Why that version specifically by default?

Reply:
no reason. inherited from old sandbox test